README.md

```diff
@@ -41,17 +41,32 @@ client = WebSocketStreamingClient(access_token="ACCESS TOKEN")
 Create a generator function which yields chunks of audio (objects supporting the `bytes-like` interface).
 The StreamingClient will use your generator as input, iterating it and sending each audio chunk to the Speech Recognition service.
 
+**Important: The Speech Recognition service expects the audio chunks to arrive at a realtime pace, or slower. Faster than realtime pace may cause the service to behave unexpectedly.**
+
 _Note:
-The current version of the service supports only WAV format (pcm_s16le - PCM signed 16-bit little-endian).
-Your generator should output audio chunks containing this format, which may also include WAV headers._
+The current version of the service supports only raw PCM format (pcm_s16le - PCM signed 16-bit little-endian).
+Your generator should output audio chunks in this format, **without** any format headers._
+
+#### End of Stream
+When the media generator is exhausted, the client should send an End-of-Stream (non-binary) message to the service. 
+The Verbit Streaming Speech Recognition service accepts non-binary messages with the following structure:
+```
+{
+   "event": str, 
+   "payload": dict
+}
+```
+The End-of-Stream message should have the `event` field set to `"EOS"`.
```

Comment:
this is too generic I think
instead, show the actual EOS message that is sent
nobody cares about our general schema for events unless they need to use it